src/pages/SignUpPage/SignupPage.jsx

```diff
@@ -1,207 +1,275 @@
-                  <button className="btn signup-action-btn" onClick={onSubmittFunc} >Signup</button>
-                </div>
-                <Link className="signup-footer" to="/login">Already have an Account <span className="material-icons-round">
-                  navigate_next
-                </span>
-                </Link>
-            </div>
-          </main>
-        </section>
-      </div>
-    </>
-  )
+            </section>
+            <section className='content'>
+                {/* <nav className='auth-nav'>
+                    <p>Not a member? Sign up now</p>
+                </nav> */}
+                <main>
+                    <div className="signup-container">
+
+                        <div className="signup-credential-container"> {/* <label>Email Address</label> */}
+                            <input placeholder="Email Address - xyz@gmail.com"
+                                onChange={
+                                    (e) => dispatch({email: e.target.value})
+                                }/>
+                        </div>
+                        <div className="signup-credential-container"> {/* <label>Password</label> */}
+                            <div className='password-holder'>
+                            <input type={passwordType}
+                                style={
+                                    {
+                                        borderColor: passwordCheckError,
+                                        outlineColor: passwordCheckError
+                                    }
+                                }
+                                onChange={
+                                    (e) => {
+                                        PasswordCheck(e.target.value);
+                                    }
+                                }
+                                name=""
+                                placeholder="Password"
+                                id=""/>{
+                                    passwordType === "password" ?
+                                        <span class="material-icons-round" onClick={()=>PasswordVisibilityHandler() }>
+                                    visibility
+                                    </span>:<span class="material-icons-round" onClick={()=>PasswordVisibilityHandler() }>
+                                visibility_off
+                                        </span>}
+                                        </div>
+                            
+                            <p className='error'> {
+                                confirmPassword.length > 0 && confirmPassword.length < 7 ? "password should be minimum 7 letter" : ""
+                            } </p>
+                            <p className='error'> {
+                                confirmPassword.length > 0 && !passwordCheckError && "Password should contain a number, alphabet & special character"
+                            } </p>
```

Comment:
Try to move these conditions outside JSX

Reply:
done